lib/ansible/modules/packaging/os/redhat_subscription.py

```diff
@@ -717,6 +727,8 @@ def main():
     autosubscribe = module.params['autosubscribe']
     activationkey = module.params['activationkey']
     org_id = module.params['org_id']
+    if activationkey and not org_id:
```

Comment:
Can this just be in `required_together`? If not, please leave a comment why.

Reply:
`required_together` means "if any one is present, both must be present", but here the relationship is one way only, if you have activation key you much provide an org, but you may also provide an org without providing an activation key. For example, `subscription-manager register --username=foo --password=blah --org=blah` is valid and `subscripton-manager register --activationkey=blah --org=foo` is also valid.

Reply:
Would be nice to see in the code as a comment... Maybe something like:

    # activation key requires org, but org does not require activation key

Not a blocker though.